src/Samples/CodeEditorContribution/CodeEditorContribution.json

```diff
@@ -0,0 +1,14 @@
+{
+    "contributions": [
+        {
+            "id": "my-code-editor-contribution",
+            "type": "ms.vss-web.action",

```

Comment:
Curious, is an `action` really the correct contribution type for this?

Reply:
I based it on the [custom content renderer sample](https://github.com/Microsoft/vsts-extension-samples/blob/master/custom-content-renderer/vss-extension.json). Is there another contribution type that would be better?

Reply:
@nkirchem Hm, before I saw anything wrong, is everything 3rd party now an action?

Reply:
No, definitely not an action here. The Code area needs to define (and use) their own custom contribution type here. I don't think there are any existing contribution types that make sense, this is so specific to code.

Reply:
It turns out I can either completely leave off the "type" property or specify some made up value and it still works. @nkirchem How is "type" intended to be consumed/enforced by the extension point? I based the extension point code on the existing content renderer extension point, so any problems here are problems there as well. @hross any thoughts?

Reply:
I found a type definition for the content renderer - it just isn't used by the sample or enforced in any way that I can tell.

Reply:
I think I've figured this out - I'll send a PR on the ADO repo shortly.